src/chart/axes3d.rs

```diff
@@ -130,9 +162,9 @@ where
             BoldPoints(self.n_labels[2]),
         );
         let kps_light = chart.get_key_points(
-            LightPoints::new(self.n_labels[0], self.n_labels[0] * 10),
-            LightPoints::new(self.n_labels[1], self.n_labels[1] * 10),
-            LightPoints::new(self.n_labels[2], self.n_labels[2] * 10),
+            LightPoints::new(self.n_labels[0] * self.light_lines_limit[0], self.n_labels[0] * self.light_lines_limit[0]),
+            LightPoints::new(self.n_labels[1] * self.light_lines_limit[1], self.n_labels[1] * self.light_lines_limit[1]),
+            LightPoints::new(self.n_labels[2] * self.light_lines_limit[2], self.n_labels[2] * self.light_lines_limit[2])
```

Comment:
This looks odd to me, shouldn't it be

```rust
LightPoints::new(self.n_labels[0], self.n_labels[0] * self.light_lines_limit[0]),
```

Otherwise we would be multiplying both arguments which wasn't the case before and also doesn't happen in the 2d version.